For whatever reason, Qt5 ships this header on Linux, it does on other
distros as well, therefore this check is incorrect and breaks build.

--- wscript
+++ wscript
@@ -91,10 +91,6 @@ def configure(conf):
         if not conf.options.no_qt5:
             autowaf.check_pkg(conf, 'Qt5Widgets', uselib_store='QT5',
                               atleast_version='5.1.0', mandatory=False)
-            if conf.check_cxx(header_name = 'QMacCocoaViewContainer',
-                              uselib      = 'QT5',
-                              mandatory   = False):
-                autowaf.define(conf, 'SUIL_WITH_COCOA_IN_QT5', 1)
 
     conf.check_cc(define_name   = 'HAVE_LIBDL',
                   lib           = 'dl',
